avm/res/aad/domain-service/main.json

```diff
@@ -5,8 +5,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.25.53.49325",
```

Comment:
Thanks for the PR @ReneHezser . Let me just soft block it with this comment until one of the maintainers had a chance to configure the AVM subscription. I'll try to do that tomorrow. The benefit - we can publish a new version without the need for an artificial update of the module code.